src/Goerli-DssSpell.t.sol

```diff
@@ -458,4 +508,494 @@ contract DssSpellTest is GoerliDssSpellTestBase {
+        assertEq(dai.balanceOf(address(rwaconduitin_008)), 0, "RWA008: Dangling Dai in input conduit after push()");
+
+        rwaurn_008.wipe(daiToPay);
+        rwaurn_008.free(1 * WAD);
+
+        (ink, art) = vat.urns("RWA008-A", address(rwaurn_008));
+        assertEq(ink, preInk, "RWA008: Bad `ink` after free()");
+        assertLt(art, 4, "RWA008: Bad `art` - larger than conversion error dust after wipe()"); // wad -> rad conversion in wipe leaves some dust
+    }
+
+    function testRWA008_OPERATOR_LOCK_DRAW_CAGE() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        // set the balance of this contract
+        hevm.store(address(rwagem_008), keccak256(abi.encode(address(this), uint256(3))), bytes32(uint256(1 * WAD)));
+        assertEq(rwagem_008.balanceOf(address(this)), 1 * WAD, "RWA008: wrong gem balance");
+        // setting address(this) as operator
+        hevm.store(address(rwaurn_008), keccak256(abi.encode(address(this), uint256(1))), bytes32(uint256(1)));
+        assertEq(rwaurn_008.can(address(this)), 1, "RWA008: address(this) not operator");
+
+        rwagem_008.approve(address(rwaurn_008), 1 * WAD);
+        rwaurn_008.lock(1 * WAD);
+        assertEq(dai.balanceOf(address(rwaconduitout_008)), 0, "RWA008: dai balance is not zero for output conduit");
+        rwaurn_008.draw(1_000_000 * WAD);
+
+        (uint256 ink, uint256 art) = vat.urns("RWA008-A", address(rwaurn_008));
+        assertEq(ink, 1 * WAD, "RWA008: wrong ink in urn");
+        assertEq(art, 1_000_000 * WAD, "RWA008: wrong art in urn"); // rate == 1 RAY (drip was never called and passed 0 seconds after init)
+
+        // wards
+        giveAuth(address(rwaconduitout_008), address(this));
+        // can
+        hevm.store(address(rwaconduitout_008), keccak256(abi.encode(address(this), uint256(1))), bytes32(uint256(1)));
+        assertEq(rwaconduitout_008.can(address(this)), 1, "RWA008: output conduit not operator");
+        // may
+        hevm.store(address(rwaconduitout_008), keccak256(abi.encode(address(this), uint256(6))), bytes32(uint256(1)));
+        assertEq(rwaconduitout_008.may(address(this)), 1, "RWA008: output conduit not mate");
+
+        assertEq(dai.balanceOf(address(rwaconduitout_008)), 1_000_000 * WAD, "RWA008: wrong dai balance output conduit");
+
+        rwaconduitout_008.pick(address(this));
+
+        rwaconduitout_008.push();
+
+        assertEq(dai.balanceOf(address(this)), 1_000_000 * WAD, "RWA008: wrong dai balance address(this)");
+
+        giveAuth(address(end), address(this));
+        end.cage();
+        end.cage("RWA008-A");
+
+        end.skim("RWA008-A", address(rwaurn_008));
+
+        (ink, art) = vat.urns("RWA008-A", address(rwaurn_008));
+        assertEq(ink, 1 * WAD - (1_000_000 * WAD * WAD / 30_043_520_665599336150000000), "RWA008: wrong ink in urn after skim");
```

Comment:
Maybe point out that the expression above is the the equivalent of:

```solidity
(1_000_000 * WAD) / (30_043_520_665599336150000000 * WAD)
```

However we multiply first to avoid rounding issues.

It may seem obvious for someone more experienced, but it can look confusing.